### PR TITLE
Add benchmarking items and bench darwinia-deposit

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: runtime/crab/src/lib.rs [hunk 1]
```diff
@@ -189,9 +194,6 @@ frame_benchmarking::define_benchmarks! {
 	[pallet_vesting, Vesting]
 	[pallet_session, SessionBench::<Runtime>]
 	[pallet_timestamp, Timestamp]
-	// Bridge
-	[pallet_bridge_grandpa, BridgePolkadotGrandpa]
-	[pallet_fee_market, DarwiniaFeeMarket]
 }
```
Comment (boundless-forest) [inline runtime/crab/src/lib.rs:197]: Any particular reason for renaming `mod.rs` to `pallets.rs`? Look like mod is a better name

Comment (AurevoirXavier) [reply runtime/crab/src/lib.rs:197]: Too many files in the `pallet` folder.

It's not clearly to check mod.rs.

Comment (boundless-forest) [reply runtime/crab/src/lib.rs:197]: LGTM, after switching to your branch.

**Review by boundless-forest — DISMISSED** [review]

**Comment by AurevoirXavier** [comment]
@boundless-forest something wrong with the EVM test.

**Pull request merged**